test/fixtures/r-httr2/bash_redirect.r

```diff
@@ -0,0 +1,21 @@
+library(httr2)
+
+params = list(
```

Comment:
Why put these in their own variables instead of just inline?

Reply:
Copy-paste from the version for httr. But I inlined them now.